packages/webapp/pages/sources/[source].tsx

```diff
@@ -64,17 +64,18 @@ import HorizontalFeed from '@dailydotdev/shared/src/components/feeds/HorizontalF
 import { IconSize } from '@dailydotdev/shared/src/components/Icon';
 import { ActiveFeedNameContext } from '@dailydotdev/shared/src/contexts/ActiveFeedNameContext';
 import CustomAuthBanner from '@dailydotdev/shared/src/components/auth/CustomAuthBanner';
+import type { GraphQLError } from '@dailydotdev/shared/src/lib/errors';
 import Custom404 from '../404';
 import { defaultOpenGraph, defaultSeo } from '../../next-seo';
 import { mainFeedLayoutProps } from '../../components/layouts/MainFeedPage';
 import { getLayout } from '../../components/layouts/FeedLayout';
 
-type SourcePageProps = { source: Source };
+type SourcePageProps = { source?: Source };
 
 const SourceRelatedTags = ({
   sourceId,
 }: {
-  sourceId: string;
+  sourceId: string | undefined;
```

Comment:
Can it just become `SourcePageProps`?

Reply:
Same below

Reply:
My bad, read it wrong, above is source object. But what I mean `sourceId?: string` instead of `string | undefined` 🙏

Reply:
Ah yes :) I first started with | undefined and then realized ? was easier/more explicit.